quantmsio/utils/file_utils.py

```
@@ -44,6 +45,31 @@ def extract_len(fle, header):
     return fle_len, pos
 
 
+def load_de_or_ae(path):
+    f = open(path, encoding="utf-8")
```

Ruff

Use a context manager for opening files

(SIM115)